lib/redis/connection/memory.rb

```diff
@@ -89,6 +89,9 @@ def connect_unix(path, timeout)
       def disconnect
       end
 
+      def client(command, options)
```

Comment:
I wonder if we should implement a case statement on `command` and raise an unknown command for values other than `:setname`. We could probably have `:getname` return `nil` too, to be correct. Anything else could raise until someone finds a need for them.

Could we also add a couple of tests for these, to make sure we don't break the behaviour in the future please?